README.md

```diff
@@ -210,12 +212,18 @@ const { token, appId, expiration } = await githubAppJwt({
 });
 ```
 
-But we recommend to convert the format using `openssl` before passing it to your app.
+- #### Using OpenSSL
+
+Convert the format using `openssl` before passing it to your app.
 
 ```
 openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in private-key.pem -out private-key-pkcs8.key
 ```
 
+- #### Using an Online Private Key Converter
+
+Convert quickly using the Web interface at https://private-key-converter.vercel.app
+
```

Comment:
it's the most convenient option, wanna put it on top?

Reply:
I just thought to be chill about it... I can surely lift it to the top 😃

Reply:
Make it so :rocket: